doc/Language/regexes.pod6

```diff
@@ -44,7 +44,7 @@ operators:
     $match = m/.+/; say $match; say $match.^name; # OUTPUT: «｢abc｣␤Match␤»
     $match =  /.+/; say $match; say $match.^name; # OUTPUT: «/.+/␤Regex␤»
 
-Whitespace in literal regexes is ignored insofar as the L<:sigspace>
+Whitespace in literal regexes is ignored insofar as the L<:sigspace
```

Comment:
I think that it's actually a `C<` what's missing here. Also try to fit in in the same line.

Reply:
Thanks - you're right.